GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -236,6 +236,7 @@ public FormCommit([NotNull] GitUICommands commands, CommitKind commitKind = Comm
             Unstaged.SetNoFilesText(_noUnstagedChanges.Text);
             Unstaged.FilterVisible = true;
             Staged.SetNoFilesText(_noStagedChanges.Text);
+            Staged.FilterVisible = true;
```

Comment:
Can we make this the default so we don't need to set it explicitly?

Reply:
This fails on AppVeyor and perhaps on other strange machines. See the screenshot above.

Reply:
Now (that only `Visible` is changed), it works.

Reply:
Awesome!